src/Controllers/CSSGeneratorController.php

```diff
@@ -114,6 +118,25 @@ protected function parseFamilyParameter(string $value): array
         return $result;
     }
 
+    /**
+     * Check font-display value. Value must be one of: auto, block, swap, fallback, optional.
+     *
+     * @param $fontDisplay Font-display value or null
+     * @return string|null Valid font-display css value, or null, if $fontDisplay is null or empty.
+     * @throws \InvalidArgumentException If the parameter set, but has not valid value. The message may be sent
+     *          back to the client.
+     */
+    protected function checkFontDisplay($fontDisplay): string
+    {
+        if (!isset($fontDisplay) || $fontDisplay === '') {
+            return null;
```

Comment:
The method type says that it can return only a string, so this line will trigger an error.

Since `null` and `''` values give the same result, please change the `$fontDisplay` type to `string` everywhere with default value `''` (empty string) that means that a style value isn't set.